CredentialProvider.Microsoft/Program.cs

```diff
@@ -159,8 +165,18 @@ public static async Task<int> Main(string[] args)
                     GetAuthenticationCredentialsRequest request = new GetAuthenticationCredentialsRequest(parsedArgs.Uri, isRetry: parsedArgs.IsRetry, isNonInteractive: parsedArgs.NonInteractive, parsedArgs.CanShowDialog);
                     GetAuthenticationCredentialsResponse response = await getAuthenticationCredentialsRequestHandler.HandleRequestAsync(request).ConfigureAwait(continueOnCapturedContext: false);
 
-                    multiLogger.Info($"{Resources.Username}: {response?.Username}");
-                    multiLogger.Info($"{Resources.Password}: {(parsedArgs.RedactPassword ? Resources.Redacted : response?.Password)}");
+                    string resultUsername = response?.Username;
+                    string resultPassword = parsedArgs.RedactPassword ? Resources.Redacted : response?.Password;
+                    if (parsedArgs.OutputFormat == OutputFormat.Json)
+                    {
+                        // Manually write the JSON output, since we don't use ConsoleLogger in JSON mode (see above)
+                        Console.WriteLine(JsonConvert.SerializeObject(new CredentialResult(resultUsername, resultPassword)));
```

Comment:
I'm emitting capitalized "Username"/"Password" as the keys. If you feel strongly for "username" "password", I'll switch to using CamelCasePropertyNamesContractResolver(). I don't personally have an opinion - @shubham90 in case he does / the Microsoft official style guide (which I can't seem to find) has an opinion.

Reply:
I don't have an opinion either. I also didn't find any official doc for this. I would say this is fine.